style.css

```diff
@@ -4,9 +4,18 @@
   scroll-behavior: smooth;
 }
 
+.error {
+  color: #0d151e;
+  font-weight: 400;
+  transition: opacity 0.5s ease-in-out;
+  opacity: 0;
+  width: fit-content;
+}
+
 @media only screen and (max-width: 767.9px) {
   .desknav-bar {
     display: none;
```

Comment:
Good job with the form, it works as intended. 👍  But let's apply the [HTML&CSS Best Practices ](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)since it is one of the project requirements. For example, here you can use define a class called `d-none` or `display-none` and give them as class attributes to the related HTML tag on your HTML file.

Reply:
This preoject requirement is form validation and you reviewed code from popup window. I already got feedback for refactor the code and apply DRY principle as optional which i will change but because i need to sumbit all my activites before deadline. Please review code related to validation and i will refactor popup window code once i completed all the tasks for this week. For validation code is not too long and its not repeating.

Can you clarify more about css why do i need to give display none as i add that class through js in which div getting created and removed for showing error